### PR TITLE
Added note about user count

### PR DESCRIPTION
**Pull request by JeffSchering**
This is the note:

Make sure that the new license is for a number of users that is greater than or equal to the current total number of users on your system. To find the total number of users, go to the REPORTING section of the System Console and click **Site Statistics**. The total number of users is in the **Total Users** field. The license is rejected if this value is greater than allowed by the key.

### CODE REVIEW

File: source/install/ee-install.rst [hunk 1]
```diff
@@ -48,10 +48,12 @@ Upgrading Enterprise Edition to a newer version
...
-To change an existing license key on an Enterprise Edition server to a new license key:
+Make sure that the new license is for a number of users that is greater than or equal to the current total number of users on your system. To find the total number of users, go to the REPORTING section of the System Console and click **Site Statistics**. The total number of users is in the **Total Users** field. The license is rejected if this value is greater than allowed by the key.
 
-1. In the **System Console**, go to **OTHER** > **Edition and License** > **License Key** and click "Remove Enterprise License and Downgrade Server". This clears the license from the server and refreshes the System Console. 
-2. In the **System Console**, go to **OTHER** > **Edition and License** > **License Key** and upload the new license key file.
-   
-The **Edition** and **License** sections on the page should update to show the capabilities of your new license key. 
+**To install a new license key**:
 
+1. In the OTHER section of the System Console, click **Edition and License**. The *Edition and License* page opens.
```
Comment (lindy65) [inline source/install/ee-install.rst]: "OTHER" > "**OTHER**" ?

Comment (JeffSchering) [inline source/install/ee-install.rst]: I was thinking that too, at first. But in the System Console, OTHER is not clickable; it's just a label. If it was something that you had to click in order to see its items, then **OTHER** would be correct. Does that make sense?

Comment (lindy65) [inline source/install/ee-install.rst]: Yep, makes sense, thanks :)

File: source/install/ee-install.rst [hunk 2]
```diff
@@ -48,10 +48,12 @@ Upgrading Enterprise Edition to a newer version
...
+2. Click **Remove Enterprise License and Downgrade Server**. This clears the license from the server and refreshes the System Console.
+3. Return to the *Edition and License* page and upload the new license key file.
```
Comment (jasonblais) [inline source/install/ee-install.rst]: I think you're already on the "Edition and License" page after the page refreshes

Comment (lindalumitchell) [inline source/install/ee-install.rst]: Yes, @JeffSchering I just tested removing a license, and confirmed that after refresh, I am still viewing /admin_console/license.

**Pull request merged**